### PR TITLE
update for t1 2021 (i think)

### PR DESCRIPTION
**Pull request by crispinha**

### CODE REVIEW

**Comment by crispinha** [comment]
wait nevermind i forgot to change the date in the html

File: src/deadify.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (() => {
 	function getWeek() {
 		// The uni week without factoring out holidays/breaks
-		let weekNum = getWeekOfYear(new Date()) - 28;
+		let weekNum = getWeekOfYear(new Date()) - 7;
 
 		// Mid-term break
 		if (weekNum > 5 && weekNum < 8) {
```
Comment (NickGeek) [inline src/deadify.js]: You will probably need to change these too. Just make it so the week count doesn't increase during the break weeks.

**Review by NickGeek — CHANGES_REQUESTED** [review]

File: src/deadify.js (hunk 1 above)
Comment (crispinha) [reply src/deadify.js]: yeah i do thanks

**Review by NickGeek — APPROVED** [review]

**Comment by NickGeek** [comment]
Thanks!

**Pull request merged**